docs/runner/events_create.rst

```diff
@@ -0,0 +1,37 @@
+.. _events_create_yamlref:
+
+Events Create Story Runner YAML
+...........................................
+
+'asset_name' is required from a previously created asset. The asset_id is derived and 
```

Comment:
'previously created asset' is a bit clumsy and not technically right (it might have been unshared or untracked, for example). I'd suggest:

"'asset_name' must match the arc_display_name attribute of an existing asset for which the authorised Application Registration has Event write access"

Reply:
fixed